.circleci/config.yml

```diff
@@ -349,18 +349,24 @@ jobs:
     steps:
       - attach_workspace:
           at: /tmp/workspace
-      - run: |
-          cd /tmp/workspace/distribution-scripts
-          source build.env
-          cd docker
-          make all TARBALLS=/tmp/workspace/build/ VERSION=${CRYSTAL_VERSION}
-          make smoke-all VERSION=${CRYSTAL_VERSION}
-      - run: |
-          cd /tmp/workspace/distribution-scripts
-          source build.env
-          cd docker
-          docker login -u ${DOCKER_USER} -p ${DOCKER_PASS}
-          make push VERSION=${CRYSTAL_VERSION}
+      - run:
+          name: Enable buildx for multiarch builds
+          command: docker buildx create --use
```

Comment:
Do we really need this step? We already build the 84codes variant of images with buildx and multiarch, and that didn't need this step.